installability_runner.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+get_result () {
+    local result
+    case $1 in
+        0) result="pass" ;;
+        1) result="fail" ;;
+        *) result="error" ;;
+    esac
+    echo "$result"
+}
+
+highrc=0
+. /var/tmp/mini-tps/env
+for method in "install" "update" "downgrade" "remove"; do
+    mtps-run-tests $@ --test="$method";
+    thisrc=$?
+    thisres="$(get_result $thisrc)"
+    echo "$method result: $thisres (status code: $thisrc)"
+    if [ "$thisrc" -gt "$highrc" ]; then
+        highrc="$thisrc"
+    fi
+done
+tmtresult="$(get_result $highrc)"
+if [ -n "$TMT_TEST_DATA" ]; then
+    # generate the result JSON
+    /usr/libexec/mini-tps/viewer/generate-result-json ./mtps-logs > "$TMT_TEST_DATA/result.json"
+    # copy the viewer HTML
+    cp /usr/share/mini-tps/viewer/viewer.html "$TMT_TEST_DATA/viewer.html"
+
+    cat <<FOE > "$TMT_TEST_DATA/results.yaml"
+- name: /installability
+  result: $tmtresult
+  log:
+    - ../output.txt
+    - viewer.html
```

Comment:
I [tried it](https://artifacts.dev.testing-farm.io/45396855-657e-4fbb-8196-bdc0677c3e43/) and it looks OK.
But I think we want to see the `viewer.html` instead of the `output.txt` in the `/installability` test result/output, don't we?
Per [tmt docs](https://tmt.readthedocs.io/en/stable/spec/plans.html#results-format): "The first log item is considered to be the “main” log, presented to the user by default."

```suggestion
  log:
    - viewer.html
    - ../output.txt
```

Reply:
ahhh, I was wondering about that but wasn't sure how to test it and didn't spot that doc. Yes, that's what we want. Thanks! I'll change it.